client/app/domain-operation/progress/domain-operation-progress.controller.js

```diff
@@ -78,5 +78,14 @@ angular.module('App').controller(
           this.loading = false;
         });
     }
+
+    getTranslatedContent(translationIdPrefix) {
+      const domainExtension = this.domain.split('.').pop().toLowerCase();
```

Comment:
```suggestion
      const [,domainExtension] = this.domain.split('.');
```

Reply:
Thanks! That's a good idea and its implemented